apps/ai_reports/signals.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import httpx
+from django.conf import settings
+from django.db.models import signals
+from django.dispatch import receiver
+
+from adhocracy4.comments.models import Comment
+from apps.ai_reports import tasks
+
+client = httpx.Client()
+logger = logging.getLogger(__name__)
+
+
+@receiver(signals.post_save, sender=Comment)
+def get_ai_classification(sender, instance, created, update_fields, **kwargs):
+    if getattr(settings, "XAI_ENABLE", False):
+        if getattr(settings, "XAI_API_URL", None):
+            comment_text_changed = getattr(instance, "_former_comment") != getattr(
+                instance, "comment"
+            )
```

Comment:
as far as I understand `update_fields` can be:
1. `None` - > everything will be saved
2. `A list of fields` -> only the fields in the list will be saved 
3. `Empty list` -> nothing will be saved

So we can't get rid of the check in L19 as we don't know if the text changed in case 1